pkgs/development/python-modules/class-registry/default.nix

```
@@ -15,7 +17,7 @@ buildPythonPackage rec {
     sha256 = "0zjf9nczl1ifzj07bgs6mwxsfd5xck9l0lchv2j0fv2n481xp2v7";
   };
 
-  propagatedBuildInputs = [ six ];
+  propagatedBuildInputs = [ six ] ++ lib.optional (!isPy3k) typing;
```

```suggestion
  propagatedBuildInputs = [ six ] ++ lib.optional (isPy27) typing;
```

Small nitpick: Also the arguments would need an update.